api/src/main/resources/org/openmrs/module/ugandaemrreports/reports/MER_TX_CURRENT_2019.json

```diff
@@ -0,0 +1,635 @@
+{
+    "resourceType": "MeasureReport",
+    "id": "MER-TX-Indicators",
+    "status": "complete",
+    "type": "summary",
+    "measure": "https://ugisl.mets.or.ug/Measure/mer-tx-indicators",
```

Comment:
@dbaluku can you ask @mkaye to make Measure all lower case... Why not call it "next-generation-reporting" instead of Measure

Reply:
@kayemilton this is for you, do we really need to change this